commerce-theme-minium/commerce-theme-minium/src/css/minium/components/mini_cart.scss

```diff
@@ -1,18 +1,18 @@
 .commerce-cart {
 	background-color: $color-bg;
-	top: 100%;
 	box-shadow: -20px 0 40px $color-shadow;
 	display: flex;
 	flex-direction: column;
 	max-width: 500px;
 	position: absolute;
 	right: 0;
+	top: 100%;
 	transition:
 		box-shadow $transition,
 		transform $transition;
+	height: calc(100vh - #{$frame-size});
```

Comment:
can you sort this up higher? the auto-formatter is just confused by the extra indent :)